packages/eslint-plugin/lib/rules/no-empty-interface.js

```diff
@@ -20,32 +26,61 @@ module.exports = {
       url: util.metaDocsUrl('no-empty-interface'),
       recommended: 'error'
     },
-    schema: []
+    schema: [
+      {
+        type: 'object',
+        additionalProperties: false,
+        properties: {
+          allowSingleExtend: {
+            type: 'boolean'
+          }
+        }
+      }
+    ],
+    messages: {
+      empty: 'An empty interface is equivalent to `{}`.',
+      extendsWithoutMembers:
+        'An interface declaring no members is equivalent to its supertype.'
```

Comment:
thank you for updating rule to use `messages`, can you update tests to

Reply:
Should be done